google_nest_sdm/webrtc_util.py

```diff
@@ -0,0 +1,114 @@
+# SDP Direction Constants
+DIRECTION_SENDRECV = 'sendrecv'
+DIRECTION_SENDONLY = 'sendonly'
+DIRECTION_RECVONLY = 'recvonly'
+DIRECTION_INACTIVE = 'inactive'
+
+AVAILABLE_DIRECTIONS = [DIRECTION_SENDRECV,
+                        DIRECTION_SENDONLY,
+                        DIRECTION_RECVONLY,
+                        DIRECTION_INACTIVE]
+
+
+class WebRTCUtility:
+    """
+    This class provides methods for managing WebRTC SDP.
+    """
+
+    def get_media_direction(self, sdp, kind):
+        """
+        Retrieves the direction of media tracks from the SDP based on the kind (audio/video).
+
+        Args:
+            sdp (str): The SDP content
+            kind (str): The kind of media track to check ('audio' or 'video').
+
+        Returns:
+            str: The direction of the media track. One of 'sendrecv', 'sendonly', 'recvonly', or 'inactive'.
+        """
+        # Split the SDP into lines
+        lines = sdp.splitlines()
+
+        # Variable to track if we are in the desired media section
+        in_media_section = False
+
+        # Search for the media type and direction
+        for line in lines:
+            # Check if the line is a media description line (m=)
```

Comment:
Lets not have comments that describe exactly literally what the next line of code does since that will be self explanatory.

```suggestion
        # Track if we are in the desired media section
        in_media_section = False

        for line in sdp.splitlines():
            # Check if the line is a media description line
```

If you do think its needed to have an extra comment describing what is happening then a higher level comment would be useful e.g.:  We are looking for the first media direction line ('a=') that follows the media section ('m=') for the media track type.